matrix-sdk/src/main/java/org/matrix/androidsdk/MXDataHandler.java

```diff
@@ -2106,4 +2106,8 @@ public RoomIdsListRetroCompat(String aParticipantUserId, String aRoomId) {
 
         return directChatRoomIdsList;
     }
+
+    public RoomsRestClient retrieveRoomRestClient(){
```

Comment:
This is a good change as it greatly reduces the message chains in Room.java. It really helps with readability.